user_sync/app.py

```diff
@@ -367,32 +367,55 @@ def credentials():
     pass
 
 
-@credentials.command(
-    help="Stores all sensitive fields and updates configuration files, replacing plaintext values with keys")
+@credentials.command(help="Stores all sensitive fields and updates configuration files, replacing plaintext values with keys")
+
 def store():
     """
-    Stores credentials in the configuration file
+    Stores secure credentials in the configuration file
     This is an automated process
     """
+
     credential_manager = CredentialManager()
-    click.echo("Using backend: " + credential_manager.keyring_name)
-    click.echo("You have called the store-credential command.")
+    credential_manager.store()
+    retrieve_from_credman_keys = credential_manager.retrieve_from_credman_keys()
+    retrieve_config_credentials = credential_manager.retrieve_config_credentials()
+    if retrieve_config_credentials != retrieve_from_credman_keys:
+        raise AssertionException("Values were not stored securely in configuration file. Stored values are:"
+                                 + dict(retrieve_config_credentials))
+    click.echo(f"{retrieve_config_credentials} stored securely in configuration file.")
```

Comment:
This line is odd - we should print the results in a more defined way - e.g, a for loop of some kind